// discord-musicbot/src/config/replies.js
/**
 * Messages that are used in replies on to Discord users who interact with the bot.
 */
module.exports = {
  general: {
    // reply
    noPermission: 'je hebt geen perms voor dit command :fearful:',
    // message | @user | help command
    mentionedMessage: 'Hey {}, you should try `{}` for a list of commands. :thumbsup:',
    // message
    queueEmpty: 'het lijkt erop dat de muzieklijst leeg is. :sob:',
    // reply | video title
    youtubeVideoAdded: 'Ik heb "{}" in de muzieklijst toegevoegt.',
  },

  summonCommand: {
    // reply + error
    couldntConnect: 'Ik kan niet verbinden in het spraak kanaal',
    // reply
    notInVoiceChannel: 'je moet eerst zelf in een spraak kanaal zitten voordat ik er in kan',
  },

  disconnectCommand: {
    // reply
    notConnectedToVoice: 'Ik kan niet disconnecten als ik niet geconnect ben.',
    // reply
    notInSendersChannel: 'Je moet eerst in het channel van mij zitten voordat je me kunt laten disconnecten.',
  },

  playCommand: {
    // reply | help command
    unknownPlayUrl: 'Is dit wel een URL?',
    // reply
    notConnectedToVoice: 'Ik kan niks afspelen als ik niet in een sprak kanaal zit!',
    // reply
    notInSendersChannel: 'You need to summon me to your channel first!',
  },

  pauseCommand: {
    // reply
    notPlaying: 'Ik kan de muziek niet pauzeren als ik niks aan het afspelen ben toch? :thinking:',
    // reply
    notConnectedToVoice: 'Ik kan geen muziek pauzeren as ik niet eens in een spraak kanaal zit.',
    // reply
    notInSendersChannel: 'Moet je niet eerst naar me luisteren in de channel waar ik in zit voordat je de muziek pauzeert :neutral_face:',
  },

  resumeCommand: {
    // reply
    notConnectedToVoice: 'Ik kan geen muziek hervatten als ik niet in een spraak kanaal zit!',
    // reply
    notInSendersChannel: 'Moet je niet eerst naar me luisteren in de channel waar ik in zit voordat je de muziek hervat :neutral_face:',
    // reply
    alreadyPlaying: 'Ik ben al muziek aan het spelen! :wink:',
  },

  stopCommand: {
    // reply
    notConnectedToVoice: 'Ik kan niks stoppen als ik niet in een spraak kanaal zit!',
    // reply
    notInSendersChannel: 'Je moet eerst bij mij in het spraak kanaal ziteen voordat je de muziek kan stopen',
    // reply
    alreadyStopped: 'Ik was al gestopt met spelen :sweat:',
  },

  skipCommand: {
    // reply
    notConnectedToVoice: 'Ik kan niks skippen als ik niet eens in een spraak kanaal zit',
    // reply
    notInSendersChannel: 'Je kunt de muziek niet skippen als je niet bij mij in het spraak kanaal zit.',
    // reply
    skippingSong: 'Ik heb het lietje geskipt.',
    // reply
    nothingPlaying: 'Ik kan de muziek niet skippen als ik niks aan het spelen ben',
  },

  nowPlayingCommand: {
    // reply
    notConnectedToVoice: 'Ik kan helemaal niks spelen als ik niet verbonden ben met een spraak kanaal!',
    // reply
    nothingPlaying: 'Ik ben nu niks aan het spelen.',
  },

  playlistCommand: {
    // reply
    queueIsEmpty: 'Je afspeellijst is leeg. :sob:',
    // reply
    currentPlaylist: 'Dit is nu je afspeellijst :notes:',
  },

  helpCommand: {
    // reply | help command
    unknown: 'Ik kan geen command of allias voor dat vinden probeer eens `{}`?',
    // reply
    hereYouAre: 'Dit is alles wat ik kon vinden.',
  },

  setUsernameCommand: {
    // reply
    success: ':ok_hand: Mijn username is verandert!',
    // reply + error
    error: 'Ik kan mijn naam niet veranderen',
    // reply
    invalidUsername: 'Uhh... that doesn\'t seem to be something I could name myself...',
  },

  setAvatarCommand: {
    // reply
    success: ':ok_hand: Mijn avatar is veranderd!',
    // reply + error
    error: 'Ik kan de avatar niet veranderen',
    // reply
    invalidUrl: 'Weet je zeker dat dit een bestaant url is?',
  },
};
